extensions/wikia/Search/classes/Test/Result/ResultHelperTest.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace Wikia\Search\Test\ResultSet;
+use \WikiaBaseTest as WikiaBaseTest, Wikia\Search\Result\ResultHelper;
+
+class AbstractResultSetTest extends WikiaBaseTest {
+	public function testLimitTextLength() {
```

Comment:
Let us add a few more cases. In the end, it makes sense to have this trimming in a separate class (not related to search results in any way)